### ISSUE TITLE
Scheduling notification with timepicker

### ISSUE THREAD

**Issue by Shashwat2708** [rafaelsetragni/awesome_notifications#212]
I am trying to make an alarm app where the user enters the time using a timepicker module and I want to schedule a notification for the same. How can I do so? 
Please help as this problem is now bothering me for weeks and I still can't get to it. 
I am storing the time in a local database and I know how to retrieve the value. I just can't find how to schedule a notification.

**Comment by madd-project** [comment]
It is Really easy,
Lemme show you how

**Comment by madd-project** [comment]
# Open Time Picker

```
 Future<TimeOfDay> openTimePickerDaily(BuildContext context) async {
      final TimeOfDay t =
          await showTimePicker(context: context, initialTime: TimeOfDay.now()); // this is timepicker

      if (t != null) {
        setState(() {
    dailyReminder = t.format(context);  // this is the time user has choosen, like user may have choosed 4:45 or 6:00
hour = dailyReminder.split(" ").removeAt(0).split(":").first; // this will take out the hour from the Time, 12:08 AM, (if your phone is running in the 12-hour clock)

minute =     print(dailyReminder.split(" ").removeAt(0).split(":").last); 
});
   
```

# Set Reminder Method

```
setRemainder(hour, minute) {
    AwesomeNotifications().createNotification(
        //the contents of it.
        content: NotificationContent(
            id: id, //id
            channelKey: 'reminderNotific',
            title: "Alarm",
            body: "Today Alam is working." 

        schedule: NotificationCalendar(
          // schedule
          hour: hour,
          minute: minute,
          allowWhileIdle: true,
          timeZone: AwesomeNotifications.localTimeZoneIdentifier,
        ));
}
```

Before all of it, do Inititalize the plugin inside your main.dart, If you are getting still troubles do let me know, 
here is your task, your users may have time of 24 hour clock and 12 hour clock, for them both you have to make the app, So you have to convert the Timings accordingly, like 1 in 24 clock is written as 13, 2 as 14, 3 as 15 etc.. till 12 is 12, You know it,

**Comment by Shashwat2708** [comment]
thanks for the answer..... I will try it now and let you know..... Can we also increase the duration of the notification and add a sound as I am going to make an alarm app? 
Is it possible with awesome_notifications yet?

**Comment by madd-project** [comment]
Hey Brother, 
Can you take a look at 96 - 146 line of 
https://www.github.com/madd-project/toodolee/tree/master/lib%2Fmain.dart

of the main.dart

There is written about how to add sound source, and There is the alert only once (bool) functionality so you can set it to true or false accordingly..

**Comment by madd-project** [comment]
@Shashwat2708  I have to take a good look at How to Increase the Notification Longtivity. 


Btw, take a look at the Upper comment that I just made

**Comment by Shashwat2708** [comment]
So my main objective is that the entire song is played so that it seems like an alarm...... Will this enable me to play the entire song or just a 3 second notification tone?

**Comment by madd-project** [comment]
Use @ + rafaelsetragni and try to ask him, he may know better than me, about notification longitivity, as an alarm

**Comment by Shashwat2708** [comment]
@rafaelsetragni can I play the entire song instead of just a 3 second tone, when a notification is triggered?

**Comment by rafaelsetragni** [comment]
This is possible to achieve in a native app? Because I don't know how to do it.

**Comment by Shashwat2708** [comment]
I think it is.... There are several alarm apps on play store which do thr same so it is definitely possible. I don't know if it is possible with this particular package. 

Also, does the cancel notification method has to be in same file or can I call it from another file without mentioning in the original one?

**Comment by rafaelsetragni** [comment]
Hi! Sorry for taking so long to respond to your issue on GitHub. I was focused on releasing the new 0.7.0 version of `awesome_notifications` and `awesome_notifications_fcm` and i didn't had time enough to do both. But now I can answer all your questions.

So now i'm asking you to recreate this topic using the new issue template. There's a lot of missing informations that i need to understand your problem, and all the instructions are already in the new issue template. Also, remember to check beforehand if your issue was posted by another user.

So I will automatically close all previous issues so far.
Sorry for the inconvenience and i will be waiting for your new issue request.

Thank you so much for your support!
